tests/test_experiment_rsmpredict.py

```diff
@@ -358,3 +365,113 @@ def test_fast_predict_non_numeric(self):
         }
         with self.assertRaises(ValueError):
             _ = fast_predict(input_features, modeler, df_feature_info)
+
+    def test_fast_predict_no_feature_info(self):
+        """
+        Check that ``fast_predict()`` raises an error when
+        ``df_feature_info=None`` and the ``feature_info`` attribute
+        cannot be found.
```

Comment:
Usually you want docstrings to be a one-line summary then a blank line and then a description.